tikv/pd
# Add TiDB Dashboard PR labels by scripts

Signed-off-by: Breezewish <me@breeswish.org>



### What problem does this PR solve?

- This PR uses scripts to add TiDB Dashboard labels, which gives us more control.
- It resolves a bug that labels was previously cleared mistakenly in some PRs when conditions do not meet, for example, in https://github.com/tikv/pd/pull/3834
- needs-cherry-pick labels are not added, since we discovered that mostly these cherry picks have conflicts so that we decided to make cherry pick pull requests directly, instead of using a cherry-picker to do it.



### What is changed and how it works?

Github Action

### Check List



Tests

- Manual test (add detailed scripts or steps below)

  See effects in https://github.com/breeswish/test/pull/1, which shows that labels will not be removed and the title is correctly checked.

This PR needs to be cherry-picked to each release branch, so that PRs against these branches will run this action.

### Release note

```release-note
None
```
